.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: python
-dist: bionic
+dist: focal
```

Comment:
So I'm guessing it's the "dist: focal" that's "locking" Perl from the bionic 5.26 to focal's 5.30. A bit of background in a git log comment would help here. The issue isn't of much use since it merely points out FP value changes due to the Perl version upgrade without much context.

Reply:
The commit message had the comment, the PR description did not.  Fixed.